.github/workflows/autodocs-images.yaml

```diff
@@ -0,0 +1,54 @@
+name: Build Images Reference Docs
+on:
+  workflow_dispatch:
+env:
+  WORKDIR: images
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Check out Destination Repo"
+        uses: actions/checkout@v2
+        with:
+          path: edu
+
+      - name: "Check out Images Monorepo"
+        uses: actions/checkout@v2
+        with:
+          repository: chainguard-images/images
+          path: images
+
+      - name: "Set up workdir"
+        run: mkdir -p "${{ github.workspace }}/${{ env.WORKDIR }}" && chmod 777 "${{ github.workspace }}/${{ env.WORKDIR }}"
+
+      - name: "Update the reference docs for Chainguard Images"
```

Comment:
More curious about this than anything, but could these files get generated in the `deved-autodocs` repo with a GitHub action and uploaded to a cloud storage bucket there, then downloaded as part of this workflow here?

Reply:
there are some permission challenges for generating the files inside the repo directly, because of the user running in the container image (it usesone of our distroless php images). Uploading would be possible in a separate step since the files are shared